packages/apps/docs/src/changelog.td.ts

```diff
@@ -0,0 +1,68 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+type IChangelogComplete = Record<string, IChangelogPackage>;
+
+interface IRepo {
+  name: string;
+  repo: string;
```

Comment:
the file extension should be `github.d.ts`, isn't it?

Reply:
doh.